domains/pallets/messenger/src/tests.rs

```diff
@@ -663,6 +665,74 @@ fn test_close_channel_between_chains() {
     close_channel_between_chains(&mut chain_a_test_ext, &mut chain_b_test_ext, channel_id)
 }
 
+#[test]
+fn test_update_consensus_channel_allowlist() {
```

Comment:
Would be great to add a case covering what happen if a message arrived after chain removed from allowlist